views/js/controller/testtaker/editSubject.js

```diff
@@ -18,14 +18,16 @@
  * @author Ilya Yarkavets <ilya.yarkavets@1pt.com>
  */
 
-define(['module', 'jquery','ui/lock'],
-    function(module, $, lock){
+define([
+    'jquery',
+    'ui/lock'
+], function(module, $, lock){
 
-        var editSubjectController = {
-            start : function(options){
-                $('#lock-box').each(function() {lock($(this)).register()});
-            }
-        };
-
-        return editSubjectController;
-    });
+    return {
+        start() {
+            $('#lock-box').each(function() {
```

Comment:
In such small files, we could already change everything to the es6 syntax imho. (arrow funtions, etc.)

Reply:
Agree, but since the lexical scope is used `this` and I don't know if there is multiple elements, I don't want to create any regression on a syntax update.

Reply:
do we have multiple elements with id `#lock-box`?

Reply:
I hope not, but hope isn't enough. I won't change it part of this pr without proper investigation and tests